### PR TITLE
feat(styles): info label display mode, update design [ci visual]

### LINKED ISSUE #3180: Info Label Component: Use Fiori Accent Color
#### Is this a bug, enhancement, or feature request?
enhancement

#### Briefly describe your proposal.

currently it is only possible to used Fiori's Legend colors. 
legend-colors

We need to use Fiori's Accent colors:
https://experience.sap.com/fiori-design-web/colors/#accent-colors

What is confusing? The CSS class names contain keyword "accent", but css variables containing "legend" are used.

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
current version

#### If this is a bug, please provide steps for reproducing it.
not-accent-color

### PR DESCRIPTION
**Pull request by droshev**
## Related Issue
fixes #3180 

## Description
The design has been changed since we implemented this component. Also add examples for display only mode.

#### Please check whether the PR fulfills the following requirements

1. The output matches the design specs
- [x] All values are in `rem`
- [x] RTL support
2. The code follows fundamental-styles code standards and style
- [x] only one top level `fd-*` class is used in the file
- [x] BEM naming convention is used
- [x] Variables are used, if some value is used more than twice.
- [x] Checked if current components can be reused, instead of having new code.
3. Testing
- [x] tested Storybook examples with "CSS Resources" `normalize` option 
- [x] tested Storybook examples with "CSS Resources" `unnormalize` option 
- [x] Updated tests
- [x] last commit message should have `[ci visual]` so it can trigger chromatic visual regression (e.g. `test: run chromatic visual regression [ci visual]`)
4. Documentation
- [x] Storybook documentation has been created/updated

### CODE REVIEW

File: src/styles/info-label.scss [hunk 1]
```diff
@@ -3,31 +3,34 @@
 
 $block: #{$fd-namespace}-info-label;
 
-$fd-info-label-height: 1.125rem !default;
+$fd-info-label-height: 1rem !default;
 $fd-info-label-display-height: 1rem !default;
-$fd-info-label-icon-text-spacing: 0.1875rem !default;
-$fd-info-label-border-width: 0.063rem !default;
+$fd-info-label-icon-text-spacing: 0.125rem !default;
```
Comment (InnaAtanasova) [inline src/styles/info-label.scss:8]: if `$fd-info-label-height` is now 1rem maybe remove `$fd-info-label-display-height` as they are now the same

File: src/styles/info-label.scss [hunk 2]
```diff
@@ -3,31 +3,34 @@
...
   @include fd-reset();
   @include fd-ellipsis();
 
-  border-radius: 1.125rem;
+  display: inline-flex;
+  align-items: center;
+  border-radius: 0.5rem;
+  line-height: normal;
   height: $fd-info-label-height;
   min-width: $fd-info-label-height;
```
Comment (InnaAtanasova) [inline src/styles/info-label.scss:35]: I haven't seen in the specs value for `min-width`

File: src/styles/info-label.scss [hunk 3]
```diff
@@ -81,19 +85,20 @@ $color-accents: (
   }
 
   &--display {
-    height: $fd-info-label-display-height;
+    border-color: var(--fdInfo_Label_Display_Border_Color);
+    background-color: var(--fdInfo_Label_Display_Background_Color);
+    color: var(--fdInfo_Label_Display_Text_Color);
     min-width: $fd-info-label-display-height;
```
Comment (InnaAtanasova) [inline src/styles/info-label.scss:91]: same as the other comment, I can't find anything about min-width

File: src/styles/info-label.scss (hunk 1 above)
Comment (droshev) [reply src/styles/info-label.scss]: removed

File: src/styles/info-label.scss (hunk 2 above)
Comment (droshev) [reply src/styles/info-label.scss]: removed

File: src/styles/info-label.scss (hunk 3 above)
Comment (droshev) [reply src/styles/info-label.scss]: removed

**Review by InnaAtanasova — APPROVED** [review]

**Pull request merged**